heron/executor/tests/python/heron_executor_unittest.py

```diff
@@ -11,13 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+'''heron executor unittest'''
 
 import unittest2 as unittest
 
 from heron.executor.src.python.heron_executor import ProcessInfo
 from heron.executor.src.python.heron_executor import HeronExecutor
 
-class MockPOpen:
+# pylint: disable=line-too-long
```

Comment:
Let us keep the line length to a fixed value - whatever it is today. otherwise, people will write longer lines.